Samples/GettingStarted/BinaryClassification_SentimentAnalysis/BinaryClassification_SentimentAnalysis.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
This PropertyGroup is incorrect, as the property below won't be set for Release builds.

If you want to set `<LangVersion>` to latest, it should be done unconditionally.  You can move `<LangVersion>latest</LangVersion>` to the above PropertyGroup.